Milestone3/frontend/src/App.js

```diff
@@ -0,0 +1,35 @@
+import logo from './logo.svg';
```

Comment:
So here is an example of how to use images in React. This is onw of the few things in react that is not as straightforward, since it requires a few steps:
1. place your image in the public folder in your frontend (the same place this logo.svg file is)
2. import the image with the same format as above: import [variableName] from [relative path]
3. now in your jsx your img tags will follow the format: ,< img src={[variableName]} >. make sure to include these curly braces around your variable name.
I hope this helps a bit, but if you still cant get it to work, let me know and I can help